src/card/examples/card-timeframe.js

```diff
@@ -45,10 +46,19 @@
           footer="footerConfig" filter="filterConfig" style="width: 50%">
         Card Contents
      </pf-card>
+     <label class="label-title">Loading State</label>
+     <pf-card show-spinner="dataLoading" head-title="Card Title" sub-title="Card Subtitle" show-top-border="true" filter="filterConfigHeader" style="width: 50%">
```

Comment:
I'd add "Card Contents" here, between the <pf-card> and </pf-card>, just to show that transclusion will come through when the spinner is off.

Reply:
Will do!